src/Infrastructure/Persistance/Jwt/JwtService.cs

```diff
@@ -51,7 +53,38 @@ public async Task<AccessToken> GenerateAsync(User user)
 
             var securityToken = tokenHandler.CreateJwtSecurityToken(descriptor);
 
-            return new AccessToken(securityToken);
+            return new AccessToken(securityToken: securityToken,
+                refreshToken: GenerateRefreshToken(),
+                refreshTokenExpiresIn: _siteSetting.JwtSettings.RefreshTokenValidityInDays);
+        }
+
+        public int? ValidateJwtAccessTokenAsync(string token)
+        {
+            var secretKey = Encoding.UTF8.GetBytes(_siteSetting.JwtSettings.SecretKey); // longer that 16 character
+            var encryptionkey = Encoding.UTF8.GetBytes(_siteSetting.JwtSettings.EncryptKey); //must be 16 character
```

Comment:
Please use camelcase naming convention.

Reply:
camel-case for method names?

Reply:
For variable names, in this case -> encryptionkey